libpod.conf

```diff
@@ -1,181 +0,0 @@
-# libpod.conf is the default configuration file for all tools using libpod to
```

Comment:
Should we keep the file, but just have it contain a small comment/note that its been removed as of version 1.9 and replaced by containers.conf?